modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -365,6 +365,11 @@ public String getItemClassName() {
 				return _objectDefinition.getClassName();
 			}
 
+			@Override
+			public String getItemModelName() {
+				return _objectDefinition.getShortName();
```

Comment:
I think we should unify what we want to store for the modelName. Here, we are returning the short name (aka Student for the Student custom object), however in the Objects framework, I see [here](https://github.com/liferay/liferay-portal/blob/813533b9c7404b834415408703f9f883abab75bd/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java#L1330) we are passing the nasme (C_Student).

Could we unify this?